clap_generate/README.md

```diff
@@ -1,12 +1,23 @@
+<!-- omit in TOC -->
 # clap_generate
 
+> **Shell completion generation for `clap`**
```

Comment:
Its not only shell completions  but anything.

Reply:
In theory, yes, but I wanted to keep this concrete, with what it is currently capable of, to help people understand why this is useful to them.